Show vscode extension API version in About dialog #86

jfaltermeier opened the pull request:
#### What it does
Shows vscode extension API version in About dialog

![ksnip_20210527-102518](https://user-images.githubusercontent.com/5889696/120187714-76e8c480-c215-11eb-8850-7a9a19991138.png)

closes #86

#### How to test
Build and check about dialog

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

vince-fugnitto commented:
@jfaltermeier you can use the frontend `VSXApiVersionProvider` to get the version synchronously without having to handle things in the `postContstuct` yourself as it is already handled. I created a commit so you can see the changes, I've also added the information on the `getting-started` page (it might be good to have the info available there and the `about` like you've done):

- https://github.com/eclipse-theia/theia-blueprint/commit/35ec77c450981344a5382131e602f33ac2f75f2a

jfaltermeier commented:
> @jfaltermeier you can use the frontend `VSXApiVersionProvider` to get the version synchronously without having to handle things in the `postContstuct` yourself as it is already handled. I created a commit so you can see the changes, I've also added the information on the `getting-started` page (it might be good to have the info available there and the `about` like you've done):
> 
> * [35ec77c](https://github.com/eclipse-theia/theia-blueprint/commit/35ec77c450981344a5382131e602f33ac2f75f2a)

Thanks, I cherry-picked the commit and also rebased this on #105 which also changes the welcome page

Jeeppler commented:
@jfaltermeier thanks for implementing the feature.
